script/update_query_records.rb

```diff
@@ -0,0 +1,59 @@
+#!/usr/bin/env ruby
+# frozen_string_literal: true
+
+# This is a one-time script, can be modified and reused in the future if Query
+# params change and we need to purge existing QueryRecords of the old params.
+#
+# call with `script/update_query_records.rb "true"`
+
+require(File.expand_path("../config/boot.rb", __dir__))
+require(File.expand_path("../config/environment.rb", __dir__))
+require(File.expand_path("../app/extensions/extensions.rb", __dir__))
+
+abort(<<HELP) if ARGV.length != 1
+
+  USAGE::
+
+    ruby script/update_query_records.rb "true"
+
+  DESCRIPTION::
+
+    Updates the serialized description in `query_records`.`description`
+    to match the new keys in the `Query::Filter` class.
+
+  PARAMETERS::
+
+    --help     Print this message.
+
+HELP
+
+class UpdateQueryRecords
+  def self.update_query_records(dry_run)
```

Comment:
```suggestion
  def self.update_query_records(dry_run:)
```
Consider using a kwarg for consistency with other methods. Examples:
`UserStats#refresh_all_user_stats`,  `Synonym::make_sure_all_referenced_synonyms_exist`, etc.
Cf. https://docs.rubocop.org/rubocop/cops_style.html#styleoptionalbooleanparameter